[BUG] "Can't resolve..." error when import RuleTable component from rule-components

**Describe the bug**
When you follow the example of RuleTable component usage here https://clouddot.pages.redhat.com/frontend-components/dev/rule-components/index.html and import RuleTable to your app, you will run into the build error. The issue is found for the rule-components version = 3.2.1 (latest).

**To Reproduce**
Steps to reproduce the behavior:
1. Import <RuleTable /> to any of your project modules via `import RuleTable from '@redhat-cloud-services/rule-components/RuleTable';`
2. Build your project (`npm run build` / `yarn build`)
3. You will get the error `Module not found: Error: Can't resolve './index.scss' in '/home/gkaratae/ccx/ocp-advisor/node_modules/@redhat-cloud-services/rule-components/esm/RuleTable'`. See below for the full error stack.

**Expected behavior**
There is no build error related to the rule-components library.

**Full error stack**
```
ERROR in ./node_modules/@redhat-cloud-services/rule-components/esm/RuleTable/RuleTable.js 40:0-48
Module not found: Error: Can't resolve './index.scss' in '/home/gkaratae/ccx/ocp-advisor/node_modules/@redhat-cloud-services/rule-components/esm/RuleTable'
resolve './index.scss' in '/home/gkaratae/ccx/ocp-advisor/node_modules/@redhat-cloud-services/rule-components/esm/RuleTable'
  using description file: /home/gkaratae/ccx/ocp-advisor/node_modules/@redhat-cloud-services/rule-components/package.json (relative path: ./esm/RuleTable)
    Field 'browser' doesn't contain a valid alias configuration
    using description file: /home/gkaratae/ccx/ocp-advisor/node_modules/@redhat-cloud-services/rule-components/package.json (relative path: ./esm/RuleTable/index.scss)
      no extension
        Field 'browser' doesn't contain a valid alias configuration
        /home/gkaratae/ccx/ocp-advisor/node_modules/@redhat-cloud-services/rule-components/esm/RuleTable/index.scss doesn't exist
      .ts
        Field 'browser' doesn't contain a valid alias configuration
        /home/gkaratae/ccx/ocp-advisor/node_modules/@redhat-cloud-services/rule-components/esm/RuleTable/index.scss.ts doesn't exist
      .tsx
        Field 'browser' doesn't contain a valid alias configuration
        /home/gkaratae/ccx/ocp-advisor/node_modules/@redhat-cloud-services/rule-components/esm/RuleTable/index.scss.tsx doesn't exist
      .mjs
        Field 'browser' doesn't contain a valid alias configuration
        /home/gkaratae/ccx/ocp-advisor/node_modules/@redhat-cloud-services/rule-components/esm/RuleTable/index.scss.mjs doesn't exist
      .js
        Field 'browser' doesn't contain a valid alias configuration
        /home/gkaratae/ccx/ocp-advisor/node_modules/@redhat-cloud-services/rule-components/esm/RuleTable/index.scss.js doesn't exist
      .scss
        Field 'browser' doesn't contain a valid alias configuration
        /home/gkaratae/ccx/ocp-advisor/node_modules/@redhat-cloud-services/rule-components/esm/RuleTable/index.scss.scss doesn't exist
      as directory
        /home/gkaratae/ccx/ocp-advisor/node_modules/@redhat-cloud-services/rule-components/esm/RuleTable/index.scss doesn't exist
 @ ./node_modules/@redhat-cloud-services/rule-components/esm/RuleTable/index.js 1:0-45 1:0-45 1:0-45
 @ ./src/PresentationalComponents/ClusterRecommendations/ClusterRecommendations.js 4:0-73 22:182-191
 @ ./src/PresentationalComponents/Clusters/ClusterDetails.js 13:0-86 44:38-60
 @ ./src/SmartComponents/Clusters/Clusters.js 6:9-8:59
 @ ./src/Routes.js 5:9-7:40
 @ ./src/App.js 4:0-34 25:158-164
 @ ./src/AppEntry.js 5:0-24 20:38-41
 @ container entry ./RootApp[0]
```

--- a/packages/rule-components/src/RuleTable/RuleTable.js
+++ b/packages/rule-components/src/RuleTable/RuleTable.js
@@ -11,7 +11,7 @@ import flatten from 'lodash/flatten';
 import debounce from 'lodash/debounce';
 import isEqual from 'lodash/isEqual';
 import { calculateMeta, calculateActiveFilters, createRows } from './helpers';
-export { default as style } from './index.scss';
+import './index.scss';
 
 class RuleTable extends Component {
     state = {

